src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -70,21 +68,21 @@ public BucketMetadata create(@RequestParam(value = "name", required = true) Stri
 
     @ApiOperation(value = "Gets a bucket's metadata by ID")
     @ApiResponses(value = @ApiResponse(code = 404, message = "Bucket not found"))
-    @RequestMapping(value = "/buckets/{bucketId}", method = GET)
+    @RequestMapping(value = "/{bucketId}", method = GET)
     public BucketMetadata getMetadata(@PathVariable long bucketId) {
         return bucketService.getBucketMetadata(bucketId);
     }
 
     @ApiOperation(value = "Lists the buckets")
-    @ApiImplicitParams({ @ApiImplicitParam(name = "page", dataType = "integer", paramType = "query", value = "Results page you want to retrieve (0..N)"),
-                         @ApiImplicitParam(name = "size", dataType = "integer", paramType = "query", value = "Number of records per page."),
-                         @ApiImplicitParam(name = "sort", allowMultiple = true, dataType = "string", paramType = "query", value = "Sorting criteria in the format: property(,asc|desc). " +
-                                                                                                                                  "Default sort order is ascending. " + "Multiple sort criteria are supported.") })
-    @RequestMapping(value = "/buckets", method = GET)
-    public PagedResources list(
-            @ApiParam(value = "The name of the user who owns the Bucket") @RequestParam(value = "owner", required = false) Optional<String> ownerName,
-            Pageable pageable, PagedResourcesAssembler assembler) {
-        return bucketService.listBuckets(ownerName, pageable, assembler);
+    @RequestMapping(method = GET)
+    public List<BucketMetadata> list(
+            @ApiParam(value = "The name of the user who owns the Bucket") @RequestParam(value = "owner", required = false) Optional<String> ownerName) {
+        return bucketService.listBuckets(ownerName);
     }
 
+    @ApiOperation(value = "Lists the buckets")
+    @RequestMapping(method = DELETE)
+    public void clean() {
+        bucketService.cleanAll();
```

Comment:
i created this endpoint, because i need it in the test to clean up the buckets created by the different tests, which cause a lot inconsistency in the test result. but you are right. it will be dangerous to have it.